### ISSUE TITLE
Page reloads every time app is resumed

### ISSUE THREAD

**Issue by dambusm** [mgks/Android-SmartWebView#23]
Hi, awesome work on this, I'm loving it. I just have one issue, any time the application is left and then resumed (eg. press home and relaunch the app), it does not resume to where it was before; instead it shows the splash screen again and reloads the first page. Is there any way to maintain the current loaded page and not reload?

**Comment by mgks** [comment]
Can I know what device and API you're testing it in?
The last time I checked `onResume()` method was working fine. Though, it can happen indeed if the app rests for a while, i.e. longer than usual, in the background.

Give me the details, I'll look into it.

**Comment by mgks** [comment]
When you said relaunch the app you mean from recent apps list or launching it from the menu?
(We're all learners, don't mind me)

**Comment by mgks** [comment]
Okay budy. I did the job.
It's a kind of workaround, but works pretty fine. Just remove `android:launchMode="singleTask"` from `AndroidManifest.xml` and you're good to go. I've also update the repo.

I'm closing this issue but if this solution doesn't works for you, just reopen this thread.

**Comment by dambusm** [comment]
Hi, apologies for the very late response.

I did indeed mean relaunching the app from the app launcher, not from the recent apps list (which works just fine).

I tried what you suggest but I found that it was still reloading the page when the app is launched again. For instance, if you navigate away from the index page, press the home button and then launch the app again, it reloads back to the index.

I did a bit of research and found a solution which is working well for me. I've added a pull request with the solution I found.

Thanks again for your work!
